src/has-role-or-restrict.js

```diff
@@ -54,8 +54,8 @@ export default function (options = {}) {
       }
 
       return hook.service.find({ query }, params).then(results => {
-        if (hook.method === 'get' && Array.isArray(results) && results.length === 1) {
```

Comment:
This can still also be an Array (e.g. if pagination is disabled), so I think we need to handle both cases.

Reply:
Sorry, it's been so long and I'm not all that familiar with Feathers' internals - do you mean that both `results` can be an array as well as `results.data`? Does this need to be handled in both this case and the case on line #135?

Reply:
What I mean is that the result can be either a plain array or a [pagination object](https://docs.feathersjs.com/api/databases/common.html#pagination).